Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3.5
+
+COPY . /app
+WORKDIR /app
+RUN pip install -e .
+
+ENV XSNIPPET_API_SETTINGS=/app/xsnippet_api/default.conf
+
+EXPOSE 8000
+CMD python -m xsnippet_api
```

Comment:
It's recommended to use `ENTRYPOINT` instead of `CMD` for declaring default executable.

In case of `ENTRYPOINT`, we're able to pass any additional command line argument through `docker run` to `xsnippet_api`. Also, please consider to use `exec` form instead of `shell` one.

https://docs.docker.com/engine/reference/builder/#entrypoint